[REDESIGN] Last polish (#57)
XanaduAI/qml: glossary.rst

Key Concepts
============

> Here you find a growing glossary of key concepts around the training of variational circuits.

Reviewer: ```suggestion
Here you can find a growing glossary of key concepts for quantum machine learning.
```

Reviewer: (since "variational ciruits" could already be a technical term that someone consulting a "Key Concepts" page may not know)

Author: Good point
